references/detection/coco_utils.py

```diff
@@ -219,7 +219,7 @@ def __getitem__(self, idx):
         return img, target
 
 
-def get_coco(root, image_set, transforms, mode="instances", use_v2=False):
+def get_coco(root, image_set, transforms, args, mode="instances"):
```

Comment:
Wonder how much we want these helpers to be used as "stand-alone" i.e. withou knowledge of the rest of the recipes (and in this case, `args`)

Reply:
I would keep them "stand-alone", since it also makes it easier to understand what is going on. Right now, I can't guess from the signature what exactly we are taking from `args`. Furthermore, it is also unclear why we are taking `use_v2` and `with_masks` from `args`, but `root` has to be passed in explicitly although that is just `args.data_path`.